### PR TITLE
Added ZW089B to Product File

### PR DESCRIPTION
**Pull request by SkynetHome**
Per Issue #3619

### CODE REVIEW

**Comment by SkynetHome** [comment]
lol OMG!

File: bundles/binding/org.openhab.binding.zwave/database/products.xml [hunk 1]
```diff
@@ -1128,6 +1128,15 @@
...
 			<ConfigFile>aeon/dsb45-zwus.xml</ConfigFile>
 		</Product>
+		<Product>
+			<Reference>
+				<Type>0202</Type>
+				<Id>0059</Id>
+			</Reference>
+			<Model>ZW089B</Model>
+			<Label lang="en">Door/Window Sensor</Label>
+			<ConfigFile>aeon/zw089.xml</ConfigFile>
```
Comment (cdjackson) [inline bundles/binding/org.openhab.binding.zwave/database/products.xml]: Shouldn't this be zw089b?

**Comment by SkynetHome** [comment]
The user said the file should be Identical to the zw089 so I didnt see any need in making a seperate xml. Dont you prefer to just add a Product listing. Are you going to be able to do anything with this nightmare pull. Its been a long time since I pulled and there were a number of other things that got committed.

**Comment by SkynetHome** [comment]
It appears the other one he wants is a identical too. Do you want new xml's or just product file changes?

**Comment by cdjackson** [comment]
> On 30 Dec 2015, at 19:15, Mike Van Dusen <notifications@github.com> wrote:
> 
> The user said the file should be Identical to the zw089 so I didnt see any need in making a seperate xml.

 I’m fine with just adding the reference but the XML file in the PR has the “b" on the end so something is wrong somewhere I guess…

**Comment by SkynetHome** [comment]
I dont think we are looking at the same thing. In the diff you posted above which is the only file I intended to commit the xml is zw089.xml with the Model set to ZW089B and a matching ID for the correct model. I did do it differently last night before I managed to sync my fork with the master and those changes also got committed here. But file you referenced above, the one intend to commit is correct.

**Comment by cdjackson** [comment]
Hmmmm - if I look at the “Files Changed” tab for this PR in github, there are 2 files shown - the products.xml file with the additional reference, and a complete new zw089b.xml file…...

**Comment by SkynetHome** [comment]
omg this is such a mess. I learned how to write business applications using 5 different programming languages in the last 6 months and I still cant manage a github repo to save my life!

**Comment by SkynetHome** [comment]
I will try and rebase to remove wrong commits if that doesnt work then i'll close and re-pull

**Pull request closed without merging**